diff --git a/tests/meson.build b/tests/meson.build
index 6741f8f..316cf25 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -55,11 +55,6 @@ tests = {
     'source': 'module-test.c',
     'c_args': ['-DMODULE_TYPE="plugin"'],
   },
-  'cxx-test' : {
-    'source' : 'cxx-test.cpp',
-    'include_directories' : gmoduleinc,
-    'dependencies' : [libgio_dep],
-  },
 }
 
 test_extra_programs = {
