diff -up glib-2.64.5/glib/gmacros.h.omv~ glib-2.64.5/glib/gmacros.h
--- glib-2.64.5/glib/gmacros.h.omv~	2020-08-21 01:26:15.873959446 +0200
+++ glib-2.64.5/glib/gmacros.h	2020-08-21 01:28:17.290140369 +0200
@@ -557,7 +557,7 @@
  *
  * Since: 2.60
  */
-#if    __GNUC__ > 6
+#if    __GNUC__ > 6 || defined(__clang__)
 #define G_GNUC_FALLTHROUGH __attribute__((fallthrough))
 #else
 #define G_GNUC_FALLTHROUGH
