diff -Naur glib-2.60.0/meson.build glib-2.60.0.tpg/meson.build
--- glib-2.60.0/meson.build	2019-03-04 20:06:14.000000000 +0000
+++ glib-2.60.0.tpg/meson.build	2019-03-19 21:58:50.833000000 +0000
@@ -57,7 +57,12 @@
 
 glib_prefix = get_option('prefix')
 glib_bindir = join_paths(glib_prefix, get_option('bindir'))
-glib_libdir = join_paths(glib_prefix, get_option('libdir'))
+if runtime_libdir == ''
+  glib_libdir = join_paths(glib_prefix, get_option('libdir'))
+
+else
+  glib_libdir = os.path.realpath(join_paths(get_option('libdir'), get_option('runtime_libdir'))
+endif
 glib_libexecdir = join_paths(glib_prefix, get_option('libexecdir'))
 glib_datadir = join_paths(glib_prefix, get_option('datadir'))
 glib_pkgdatadir = join_paths(glib_datadir, 'glib-2.0')
