diff -up glib-2.70.0/gio/tests/meson.build.omv~ glib-2.70.0/gio/tests/meson.build
--- glib-2.70.0/gio/tests/meson.build.omv~	2021-09-25 20:24:44.231604024 +0200
+++ glib-2.70.0/gio/tests/meson.build	2021-09-25 20:24:58.698398174 +0200
@@ -686,7 +686,7 @@ if not meson.is_cross_build() or meson.h
     objcopy_supports_add_symbol = run_command(objcopy, '--help').stdout().contains('--add-symbol')
   endif
 
-  ld = find_program('ld', required : false)
+  ld = find_program('ld.bfd', required : false)
 
   if build_machine.system() == 'linux' and objcopy.found() and objcopy_supports_add_symbol and ld.found()
     test_gresource_binary = custom_target('test5.gresource',
