--- glib-2.42.0/glib/gmacros.h.orig	2014-10-17 10:35:34.539050217 +0100
+++ glib-2.42.0/glib/gmacros.h	2014-10-17 10:37:02.616043728 +0100
@@ -63,7 +63,7 @@
 #define G_GNUC_NULL_TERMINATED
 #endif
 
-#if     (__GNUC__ > 4) || (__GNUC__ == 4 && __GNUC_MINOR__ >= 3)
+#if !defined (__clang__) && ( (__GNUC__ > 4) || (__GNUC__ == 4 && __GNUC_MINOR__ >= 3) )
 #define G_GNUC_ALLOC_SIZE(x) __attribute__((__alloc_size__(x)))
 #define G_GNUC_ALLOC_SIZE2(x,y) __attribute__((__alloc_size__(x,y)))
 #else
