diff -uraN glib-2.70.2/gio/tests/meson.build omv-glib-2.70.2/gio/tests/meson.build
--- glib-2.70.2/gio/tests/meson.build	2021-12-03 11:02:55.821885600 +0100
+++ omv-glib-2.70.2/gio/tests/meson.build	2021-12-03 17:46:45.807884809 +0100
@@ -693,7 +693,7 @@
     objcopy_supports_add_symbol = run_command(objcopy, '--help', check: false).stdout().contains('--add-symbol')
   endif
 
-  ld = find_program('ld', required : false)
+  ld = find_program('ld.bfd', required : false)
 
   if build_machine.system() == 'linux' and objcopy.found() and objcopy_supports_add_symbol and ld.found()
     test_gresource_binary = custom_target('test5.gresource',
