diff -up glib-2.64.3/gio/tests/meson.build.omv~ glib-2.64.3/gio/tests/meson.build
--- glib-2.64.3/gio/tests/meson.build.omv~	2020-05-21 14:17:18.823661348 +0200
+++ glib-2.64.3/gio/tests/meson.build	2020-05-21 14:17:40.166898353 +0200
@@ -669,7 +669,7 @@ if not meson.is_cross_build() or meson.h
 
   ld = find_program('ld', required : false)
 
-  if build_machine.system() == 'linux' and cc.get_id() == 'gcc' and objcopy.found() and objcopy_supports_add_symbol and ld.found()
+  if false and build_machine.system() == 'linux' and cc.get_id() == 'gcc' and objcopy.found() and objcopy_supports_add_symbol and ld.found()
     test_gresource_binary = custom_target('test5.gresource',
       input : 'test5.gresource.xml',
       output : 'test5.gresource',
